scripts/ios/azure_ios_build_psv.sh

```diff
@@ -26,7 +26,7 @@ if [[ ${USE_LATEST_XCODE} == 0 ]]; then
   # Due to some bug which is cmake cannot detect compiler while called
   # from cmake itself when project is compiled with XCode 12.4 we must
   # switch to old XCode as a workaround.
-  sudo xcode-select -s /Applications/Xcode_11.7.app
```

Comment:
Not big deal but looks like this mechanism got a bit misleading naming with several macos and xcode versions.
Just a comment, not something to fix in this PR